stdlib/json/encode.go

```diff
@@ -100,7 +101,7 @@ func convertValue(v values.Value) (interface{}, error) {
 		return nil, errors.New(codes.Invalid, "cannot encode a function value")
 	case semantic.Dictionary:
 		dict := v.Dict()
-		d := make(map[interface{}]interface{}, dict.Len())
+		d := make(map[string]interface{}, dict.Len())
```

Comment:
Can you make a comment about how json only allows string keys? I didn't know that before reading this: https://www.json.org/json-en.html.